module/applications/global/lp-history.mjs

```diff
@@ -0,0 +1,95 @@
+
+import LpTrackingData from "../../data/advancement/lp-tracking.mjs";
+
+/**
+ * 
+ * @param {object} inputs actor data
+ * @returns 
+ */
+export async function getLegendPointHistoryData( inputs ) {
+  return new Promise( ( resolve ) => {
+    let history = new LegendPointHistoryEarnedPrompt( inputs, resolve );
+    history.render( true );
+  } )
+}
+
+/**
+ * The application responsible for handling Legend Point History of Earned Points
+ *
+ * @augments {FormApplication}
+ *
+ * @param {LegendPointHistoryEarnedPrompt} legendpointHistoryEarned         The data model which is the
+ *      target data structure to be updated by the form.
+ * @param {FormApplicationOptions} [options={}]     Additional options which
+ *      modify the rendering of the sheet.
+ */
+export default class LegendPointHistoryEarnedPrompt extends FormApplication {
```

Comment:
Why is this a Form Application againg? What exactly should be editable? Only the description, right?